.travis.yml

```diff
@@ -3,15 +3,17 @@ language: python
 addons:
     apt:
         packages:
-            - bwa
             - cloc
+            - git
 branches:
   only:  # don't build "pushes" except on the master branch
     - master
 cache: pip
 python:
     - 3.5
     - 3.6
+before_install:
+    - git clone https://github.com/lh3/bwa.git && cd bwa && make -j 2 && sudo cp bwa /usr/local/bin && cd -
```

Comment:
This is the first time I've encountered a scenario where the BWA version made any difference. Time to update the CI config.